packages/cli/src/models/files/NetworkFile.ts

```diff
@@ -37,9 +37,8 @@ interface SolidityLibInterface {
 }
 
 export interface ProxyInterface {
-  alias?: string;
+  contractName?: string;
```

Comment:
As with the `ProjectFile`, if we are changing the scheme of the JSON files, we need to bump the `manifestVersion` and provide an automatic migration. Let's make sure we collect all breaking changes we are doing.